packages/manager/modules/telecom-dashboard/src/identity-check/form/telecom-dashboard-identity-check-form.module.js

```diff
@@ -0,0 +1,34 @@
+import angular from 'angular';
+
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-ui-router-title';
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@ovh-ux/manager-telecom-styles';
+import 'angular-translate';
+import 'angular-ui-validate';
+
+import component from './telecom-dashboard-identity-check-form.component';
+import routing from './telecom-dashboard-identity-check-form.routing';
+import service from '../telecom-dashboard-identity-check.service';
+
+import cancellation from './cancellation';
+
+const moduleName = 'ovhManagerTelecomDashboardIdentityCheckForm';
+
+angular
+  .module(moduleName, [
+    cancellation,
+    'ngUiRouterTitle',
+    'ovhManagerCore',
+    'pascalprecht.translate',
+    'ngOvhTelecomUniverseComponents',
+    'ui.router',
+    'ui.validate',
+  ])
+  .config(routing)
+  .component('identityCheckForm', component)
```

Comment:
Not too generic ?

Reply:
The module is the form for the identity-check. I don't see how to put something more precise.